init/jstParser.js

```diff
@@ -0,0 +1,81 @@
+class JstParser {
+  constructor(jst, settings = {}) {
+    this.jst = jst;
+    let { params, conditions, loops } = settings;
+    !conditions ? (this.conditions = []) : (this.conditions = conditions);
+    !params ? (this.params = []) : (this.params = params);
+    !loops ? (this.loops = []) : (this.loops = loops);
+  }
+  parseLoops() {
+    if (this.loops.length) {
+      let js = '';
+      let parts = this.jst.split('-<f-');
+      let i = 0;
+      parts.forEach((part, indexF) => {
+        if (part.indexOf('-f>-') + 1) {
+          let [loop, content] = part.split('-f>-');
+          const oneLoop = this.loops[i];
+          i++;
+          oneLoop.forEach((loopParams, index) => {
+            let tmp = '' + loop;
+            loopParams.forEach((param, pIndex) => {
```

Comment:
Nested x3 forEach is not a good practice. Better use for..of here and decompose to functions to avoid nested loops.